### PR TITLE
Responsive fixes

### LINKED ISSUE #133: Allow for graceful window resizing across various size groups
Similar to bootstrap xs, sm, md, lg col grouping.  When width degrades past certain px range then re-render to accommodate new size

### PR DESCRIPTION
**Pull request by Jujhar**
This is the second and complete set of responsive fixes for different smaller screen sizes.

Please review and list any imperfections and improvements you would like to see with this update and I will revise. 

Partially addresses #133.

### CODE REVIEW

**Comment by alexlyp** [comment]
Please run `lint-fix` and address the issues it shows you.  

Also please revert changes to .gitignore

**Comment by Jujhar** [comment]
Revised lint and .gitignore as requested.

I removed vs from .gitignore, how come though? 
It was just the Visual Studio project folder.

**Comment by alexlyp** [comment]
Well it's 2 things.  I dont like random things being added to ignore just to suit someones personal set up.  Plius, it's extraneous to the work you are trying to fix.  If you think it's a worthy change, open an issue and make a separate PR for it and we can discuss the merits of it there.

**Comment by Jujhar** [comment]
No need does not make me much of a difference.

File: .gitignore [hunk 1]
```diff
@@ -12,4 +12,4 @@ bin/
 .vagrant
 dist/
 app/i18n/extracted/app
-/*.node
+/*.node
```
Comment (alexlyp) [inline .gitignore]: please redo the last line break please

**Comment by Jujhar** [comment]
Updated, line break added

(I was prudish about that, I did go git checkout and grab the file just to be sure it was exactly the same but then had to checkout to a different commit so when I did it a  second time / referred file  I copied and pasted the raw text from GitHub which I guess removed the line break sorry, generally I guess files should have line breaks at end I guess)

**Pull request merged**